.github/workflows/report_refresh.yml

```diff
@@ -7,6 +7,17 @@ on:
     - cron: '0 16 * * *'
 
 jobs:
+  hubspot:
+    name: "Import Hubspot data into the report's DB"
+    uses: hypothesis/workflows/.github/workflows/eb-task.yml@main
+    with:
+      App: ${{ github.event.repository.name }}
+      Env: 'prod'
+      Timeout: 3600
+      Region: 'us-west-1'
+      Command: 'python bin/hubspot_import.py'
+    secrets: inherit
+
   refresh:
     name: "Update report information"
     uses: hypothesis/workflows/.github/workflows/eb-task.yml@main
```

Comment:
Should we add a `needs: hubspot` here to always have the hubspot data in before? or would GHA never palatalize them if not told explicitly?